rendered/manifests/otel-logs/configmap-agent.yaml

```diff
@@ -232,6 +232,7 @@ data:
           type: move
         poll_interval: 200ms
         start_at: beginning
+        storage: file_storage
```

Comment:
Curious why it needs to be added now explicitly? Didn't it work before?

Reply:
This is related to https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/13418